scripts/get-oras.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+# This script find the release by tag, downloads it to the user's home directory
+# extracts the binary to a folder in the user's home directory
+# then tries to link that binary to /usr/local/bin, if it fails to do that
+# it will try to link to $GOPATH/bin instead
+
+# Logging
+LOGFILE=get-oras.sh.log
+RETAIN_NUM_LINES=10
+
+logsetup() {
+    TMP=$(tail -n $RETAIN_NUM_LINES $LOGFILE 2>/dev/null) && echo "${TMP}" >$LOGFILE
+    exec > >(tee -a $LOGFILE)
+    exec 2>&1
+}
+
+log() {
+    echo -e "[$(date --rfc-3339=seconds)]: $*"
+}
+
+failure() {
+    local RED='\033[0;31m'
+    local NC='\033[0m' # No Color
+    log "${RED}$*${NC}"
+}
+
+notice() {
+    local CYAN='\033[0;36m'
+    local NC='\033[0m' # No Color
+    log "${CYAN}$*${NC}"
+}
+
+# Setup logging for troubleshooting
+logsetup
+
+# Search Parameters
+REPO='oras-project/oras'
+TAG='v0.11.0' # TODO: This tag will need to change later
+OS='linux'    # TODO: Can add OS/Architecture detection later
```

Comment:
How to override those environment variables?

Reply:
So the way to do this now is: 
`REPO='juliusl/oras' TAG='v0.11.5-alpha' ./get-oras.sh;. ./install-oras.sh`

or from the repo 

`curl https://raw.githubusercontent.com/juliusl/oras/juliusl/prototype-3/scripts/get-oras.sh | REPO='juliusl/oras' TAG='v0.11.5-alpha'  sh;. ./install-oras.sh`